packages/desktop-client/src/components/mobile/MobileNavTabs.tsx

```diff
@@ -265,12 +267,14 @@ function NavTab({ Icon: TabIcon, name, path, style, onClick }: NavTabProps) {
         flexDirection: 'column',
         textDecoration: 'none',
         textAlign: 'center',
+        textWrap: 'balance',
         userSelect: 'none',
+        justifyContent: 'center',
```

Comment:
Sorry, I'm really nitpicking now, but this centres the buttons within the lower div of the nav tab, which puts it lower than the true centre point when you include the grab handle.

Is it needed for spacing when the text wraps or could it be removed?

If it's needed then a negative margin of about -6 or -7px could be added here to recentre it here 
https://github.com/actualbudget/actual/pull/5429/files#diff-d13f9632e3584faa5579f2064dbf9383cc1f7bebfaedd9f98789e67cd289e6f9R229-R238

Reply:
A nitpick a day keeps the ~~bugs~~ **cough** features away 😄

It is not needed for anything related to the original bug report. Removed it 👍🏼